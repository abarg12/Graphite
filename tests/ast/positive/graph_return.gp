graph <int, tree, connected> sometraversal(graph<int, flag1, flag3> somevar) {
    graph<int, tree, blah> pants;
    return pants;
}