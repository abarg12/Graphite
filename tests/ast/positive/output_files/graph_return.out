graph <int> sometraversal(graph <int> somevar) 
{
graph <int> pants;
return pants;
}
